exercises/etl/canonical-data.json

```diff
@@ -10,7 +10,10 @@
         "such that keys are integers. e.g. in JavaScript, it might look ",
         "like `transform( { 1: ['A'] } );`"
       ],
-      "description": "transforms the a set of scrabble data previously indexed by the tile score to a set of data indexed by the tile letter",
+      "description": "Transform legacy to new",
+      "comments": [ "Transforms a set of legacy scrabble data stored as letters per score",
```

Comment:
There are now two comments fields on the same level.

Reply:
Oh woops. Missed that in the diff.

Reply:
@bencoman The new comment is great. Could you perhaps move it to the existing comment on the same level?